### DISCUSSION TITLE
MDX_Tool 1.2

### DISCUSSION THREAD

**Discussion by BobanSpasic** [BobanSpasic/MDX_Tool#3] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/BobanSpasic/MDX_Tool/releases/tag/1.2.0.12'>MDX_Tool 1.2</a>.</em>

**Comment by Banana71** [comment]
Hi Boban, 
I can't download your MDX_Tool. My AntiVirus scanner responds. "Trojan:Script/Wacatac.H!ml" ??

**Reply by BobanSpasic** [reply]
Please try downloading again. I've compiled it with another compiler flags now.
Here are the antivirus test results:
https://www.virustotal.com/gui/file/d1a478a3d66e1657a2bd960be7460488836b45c579cdf50d66d3595ea21331c1?nocache=1

Sources also updated with the new compiler flags included.

**Reply by BobanSpasic** [reply]
btw. I just found on the internet that Windows Defender got some bad definitions some 2-3 weeks ago, where most of the packed files (e.g. zip files) were marked as Trojan:Script/Wacatac.H!ml

**Reply by Banana71** [reply]
> btw. I just found on the internet that Windows Defender got some bad definitions some 2-3 weeks ago, where most of the packed files (e.g. zip files) were marked as Trojan:Script/Wacatac.H!ml

![av](https://user-images.githubusercontent.com/104296205/231676830-30353898-5016-4b3a-8d93-55699fbd6a08.png)

**Reply by BobanSpasic** [reply]
I have added an unpacked EXE to the release. Could you try to download that one?
